src/Systems/MultiplayerSystem.h

```diff
@@ -92,4 +113,8 @@ class MultiplayerSystem : public System
     const std::unordered_map<uint32_t, Entity>& getPlayers();
     const std::vector<MultiplayerDungeonUpdate>& getRemoteDungeonUpdates();
     comm::InitialInfo registerPlayer(const Entity player);
+    std::string addMessageSize(const std::string& serializedMsg);
+
+    static constexpr int m_frames = 2000;
+    static constexpr int m_enemySpeed = 50;
```

Comment:
Po co public static? Jeżeli ma być to używane w innych miejscach niż ten system, to powinno być w configu albo innych globalach